components/gridlayout/overview.md

```diff
@@ -0,0 +1,160 @@
+    </GridLayoutColumns>
+    <GridLayoutRows>
+        <GridLayoutRow Height="50px"></GridLayoutRow>
+    </GridLayoutRows>
+    <GridLayoutItems>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 1
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 2
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 3
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 4
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 5
+            </div>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+````
+
+>caption The result from the code snippet above
+
+![overview of the GridLayout functionality](images/gridlayout-overview-basic-example.png121)
+
+
+>caption Component namespace and reference
+
+````CSHTML
+<TelerikGridLayout @ref="@GridLayoutRef">
+    <GridLayoutColumns>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+    </GridLayoutColumns>
+    <GridLayoutRows>
+    </GridLayoutRows>
+    <GridLayoutItems>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 1
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 2
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 3
+            </div>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+
+@code {
+    Telerik.Blazor.Components.TelerikGridLayout GridLayoutRef { get; set; }
+}
+````
+
+
+## Features
+
+The StackLayout offers the following features:
+
+* `Class` - the CSS class that renders on the main wrapping element of the component.
+
+* `Width`- takes a CSS unit that determines how wide the GridLayout is. See the [Dimensions]({%slug common-features/dimensions%}) article for more details on what units you can use and how dimensions in percent work.
+
+* `ColumnSpacing` - controls the space between the columns in the GridLayout. See the [Dimensions]({%slug common-features/dimensions%}) article for more details on what units you can use and how dimensions in percent work.
+
+* `RowSpacing` - controls the space between the rows in the GridLayout. See the [Dimensions]({%slug common-features/dimensions%}) article for more details on what units you can use and how dimensions in percent work.
+
+* `HorizontalAlign` - controls the alignment of the inner items in the GridLayout column based on the X axis. Takes a member of the `GridLayoutHorizontalAlign` enum:
```

Comment:
Which are the inner items and which are the "outer" items?

Reply:
The inner items are all <GridLayoutItem>s defined. They are inner to the component